src/lib/y2partitioner/actions/controllers/md.rb

```diff
@@ -249,8 +249,7 @@ def available?(partition)
           return false if partition.md
           return true if partition.filesystem.nil?
 
-          mount = partition.filesystem.mountpoint
-          mount.nil? || mount.empty?
+          partition.filesystem.mount_point.nil?
```

Comment:
Is it possible that we have a mount point, but that mount point doesn't have a path or the path is empty? What would happen in that case?